pom.xml

```diff
@@ -137,6 +241,32 @@
         </dependency>
 
         <!-- test dependencies -->
+        <dependency>
```

Comment:
Nit: Maybe we should move them out of the test dependency section?

Reply:
Good point, slightly restructured them.. 👍